src/storage/admin/AdminTaskManager.cpp

```diff
@@ -135,13 +135,12 @@ void AdminTaskManager::handleUnreportedTasks() {
                                     jId,
                                     tId,
                                     apache::thrift::util::enumNameSafe(rc));
-        if (rc == nebula::cpp2::ErrorCode::E_LEADER_CHANGED ||
-            rc == nebula::cpp2::ErrorCode::E_STORE_FAILURE) {
-          ifAnyUnreported_ = true;
-          continue;
-        } else {
+        if (rc == nebula::cpp2::ErrorCode::SUCCEEDED) {
           keys.emplace_back(key.data(), key.size());
           break;
```

Comment:
No need to break here? There may be more than one future?

Reply:
yes, fixed